lib/slack-ruby-bot-server/app.rb

```diff
@@ -2,7 +2,6 @@ module SlackRubyBotServer
   class App
```

Comment:
Should break this up and out into database-specific classes. For example you could have an `Adapter` class with a common interface, and `SlackRubyBotServeR::Config.database_adapter` would no longer be an enum, but an actual instance of that thing that you can call methods like `check!` and `prepare!`.